### ISSUE TITLE
Error: Unauthorized, "Invalid Cookie"

### ISSUE THREAD

**Issue by mariuseikenes** [luizjr92/stock-info#21]
Hi - I've been using this lib for quite some time now, however recently it completely stopped working and gives a really long error.  

Is this a change in yahoo's API? Or am I being rate-limited/IP-banned? 
Please let me know if you need more info.

**Comment by patrocinio** [comment]
It seems the Yahoo API changed:
```
curl "https://query1.finance.yahoo.com/v7/finance/quote?symbols=AACIW"
{"finance":{"result":null,"error":{"code":"Unauthorized","description":"Invalid Cookie"}}}
```

**Comment by patrocinio** [comment]
Some discussion here: https://stackoverflow.com/questions/62966122/yahoo-finance-cookie-not-valid

**Comment by mariuseikenes** [comment]
Interesting. Will this be fixed in the lib? Can it be fixed?

**Comment by luiz-chagas** [comment]
Looking into it, thank you for bringing this up

**Comment by mariuseikenes** [comment]
Seems like it might've just been a bug from Yahoo - It works now.

**Comment by mariuseikenes** [comment]
The issue is back again

**Comment by patrocinio** [comment]
The yahoo-finance2 npm (https://www.npmjs.com/package/yahoo-finance2) seems to be working fine. Switching to it...

**Comment by luiz-chagas** [comment]
Closed by #22
